jenkins/couchbase-lite-core-black-duck-manifest.yaml

```diff
@@ -1,48 +1,47 @@
 meta:
```

Comment:
I alphabetized this file, and added libicu. I also deleted zlib since we don't seem to ship it, except probably we do on Windows? It's a bit confusing.